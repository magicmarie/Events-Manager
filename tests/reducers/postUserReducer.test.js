import React from "react";
import { shallow } from "enzyme/build";
import RegisterComponent from "../../src/components/Auth/Signup";

 describe("register component", () => {
  let wrapper;
  let props = {
    username: "",
    email: "",
    password: "",
    confirmPassword: "",
    errors: {
      username: "",
      email: "",
      password: ""
    },
    handleOnChange: jest.fn(),
    handleOnSubmit: jest.fn()
  };
  beforeEach(() => {
    wrapper = shallow(<RegisterComponent/>);
  });
  it("should render without fail", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
