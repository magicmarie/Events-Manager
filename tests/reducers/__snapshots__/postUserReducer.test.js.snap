// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`register component should render without fail 1`] = `ShallowWrapper {}`;
